Library/Homebrew/requirements/perl_requirement.rb

```diff
@@ -0,0 +1,28 @@
+class PerlRequirement < Requirement
+  fatal true
+  default_formula "perl"
+
+  def initialize(tags)
+    @version = tags.shift if /(\d.\d{1,2})/ === tags.first
```

Comment:
> You need to escape the dot in the regular expression: . ➡️ \..

👍 Nice spot, I missed that. 

> Aside from that, is there a reason not to make that /^\d+\.\d+$/?

To be honest, I mostly ripped this requirement straight out of the Ruby one because they serve the same purpose and now the `keg_only` status isn't a problem it didn't need the "special touches" of the previous one.

The change of regex from `/(\d\.)+\d/` in Ruby to `/(\d.\d{1,2})/` here is because we care about the difference between say Perl 5.18 and Perl 5.19, whereas the Ruby Req only cares about the `2.3` of `2.3.1` and such. 

It also felt relatively safe to stick to the two digits because a Perl version `x.100.x` seems unlikely 😄. Your suggestion seems fine, poked it locally.